package-examples/nginx/Kptfile

```diff
@@ -5,5 +5,3 @@ metadata:
 pipeline:
   validators:
   - image: gcr.io/kpt-fn/kubeval:v0.1
-    configMap:
```

Comment:
why remove this? It's need to ensure incorrect field names are caught

Reply:
But this is failing with error `ERR  - stdin: Failed initializing schema file:///tmp/master-standalone-strict/service-v1.json: open /tmp/master-standalone-strict/service-v1.json: no such file or directory`

It does basic validations without this flag which is good enough for the scope of this guide IMO. It would be friction for users if they run into this.

Reply:
It seems it works for me. Please ensure you have run `docker pull gcr.io/kpt-fn/kubeval:v0.1`. You might encounter  https://github.com/GoogleContainerTools/kpt/issues/1798